docs/release_notes/v0.11.3.md

```diff
@@ -0,0 +1,22 @@
+  
+# Dapr 0.11.3
+
+## Fixes
+
+* Improve actor service health check relibility to avoid unexpected placement disconnection(https://github.com/dapr/dapr/pull/2292).
+* Do not track Actor ID for actor pending lock count metric(https://github.com/dapr/dapr/pull/2295)
+
+### Overview
+
+We have run load test for actor to evaluate the performance and relibility with the load test since the last milestone. The test discovered [the below issues](https://github.com/dapr/dapr/issues/2093) causing memory leak:
```

Comment:
Suggest change this to "Actor load tests were run to evaluate the performance and relibility. The test discovered [these issues](https://github.com/dapr/dapr/issues/2093) that cause memory leaks"

Reply:
fixed.